modules/util/url.go

```diff
@@ -52,7 +52,11 @@ func IsExternalURL(rawURL string) bool {
 	if err != nil {
 		return true
 	}
-	if len(parsed.Host) != 0 && strings.Replace(parsed.Host, "www.", "", 1) != strings.Replace(setting.Domain, "www.", "", 1) {
+	host := setting.Domain
```

Comment:
Shouldn't this also use parsed root url?

Reply:
Yeah. Sorry I was making these changes based on your comments. I'll pop another one up.

Reply:
```suggestion
	appURL, _ := url.Parse(setting.AppURL)
```